nginx-auth0/nginx.conf

```diff
@@ -155,12 +160,24 @@ http {
          if res.email then
            ngx.log(ngx.DEBUG, "bearer_jwt_verify response: mail ", res.email)
            ngx.req.set_header("X-Akvo-Email", res.email)
+           ngx.var.akvoemail = res.email
          else
            ngx.say(cjson.encode({error="Invalid access_token"}))
            ngx.exit(ngx.HTTP_FORBIDDEN)
 	 end
       }
 
+      proxy_cache http_204;
+      proxy_cache_revalidate off;
+      proxy_cache_background_update on;
+      proxy_cache_lock on;
```

Comment:
Love it! It makes me wonder if we should look at this (or similar) for the whole API (as another Github issue).

Can we set also `proxy_cache_lock_timeout` and `proxy_cache_lock_age`? I think the defaults are too low for the latency that I expect the sync API will have if there is any data to be fetched.

Reply:
> It makes me wonder if we should look at this (or similar) for the whole API (as another Github issue). - https://github.com/akvo/akvo-flow-api/issues/204

Reply:
> Can we set also `proxy_cache_lock_timeout` and `proxy_cache_lock_age`

Yes